CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [Unreleased]
+
+#### Warning - Potentially breaking change
+- Restivus used to store the account login token in the user document: `services.resume.loginTokens.token`
```

Comment:
No problem. I updated it with modification.  (Their token is invalid, but their credentials are safe)